openassessment/xblock/static/js/src/lms/oa_staff_area.js

```diff
@@ -310,30 +380,71 @@
         },
 
         /**
-         * Submit the staff assessment.
+         * Submit the staff assessment override.
          *
          * @param {string} submissionID The ID of the submission to be submitted.
          * @param {element} rubric The rubric element to be assessed.
+         * @param {element} scope An ancestor element for the submit button (to allow for shared
+         *     classes in different form).
          */
-        submitStaffAssessment: function(submissionID, rubric) {
-            // Send the assessment to the server
+        submitStaffOverride: function(submissionID, rubric, scope) {
             var view = this;
-            var baseView = this.baseView;
-            baseView.toggleActionError('staff', null);
-            view.staffSubmitEnabled(false);
-
-            this.server.staffAssess(
-                rubric.optionsSelected(), rubric.criterionFeedback(), rubric.overallFeedback(), submissionID
-            ).done(function() {
+            var successCallback = function() {
                 // Note: we ignore any message returned from the server and instead
                 // re-render the student info with the "Learner's Final Grade"
                 // section expanded. This section will show the learner's
                 // final grade and in the future should include details of
                 // the staff override itself.
                 view.loadStudentInfo({expanded_view: 'final-grade'});
-            }).fail(function(errorMessage) {
-                $('.staff-override-error').html(_.escape(errorMessage));
-                view.staffSubmitEnabled(true);
+            };
+            this.callStaffAssess(submissionID, rubric, scope, successCallback, '.staff-override-error');
+        },
+
+        /**
+         * Submit the staff grade, and check out another learner for grading if continueGrading is true.
+         *
+         * @param {string} submissionID The ID of the submission to be submitted.
+         * @param {element} rubric The rubric element to be assessed.
+         * @param {element} scope An ancestor element for the submit button (to allow for shared
+         *     classes in different form).
+         * @param {boolean} continueGrading If true, another learner will be marked as "In Progress",
+         *     and a new grading form will be rendered with the learner's answer.
+         */
+        submitStaffGrade: function(submissionID, rubric, scope, continueGrading) {
+            var view = this;
+            var successCallback = function() {
+                view.staffGradeFormLoaded = false;
+                var showFullGradeTab = function() {
+                    // Need to show the staff grade component again, unfortunately requiring a global selector.
```

Comment:
This code is removed in #825, so that we just re-render the counts template and the grade override form.